scm/etc/scripts/MOSAiC_SS_forcing_file_generator.py

```diff
@@ -0,0 +1,590 @@
+#!/usr/bin/env python
```

Comment:
@hertneky Everything that I added to the MOSAiC_AMPS_forcing... file, I repeated here.